docs/package.json

```diff
@@ -42,7 +42,7 @@
     "@mdx-js/mdx": "^1.6.16",
     "@mdx-js/react": "^1.6.16",
     "@philpl/buble": "^0.19.7",
-    "@rasahq/docusaurus-theme-tabula": "^0.7.8",
+    "@rasahq/docusaurus-theme-tabula": "^0.7.11",
```

Comment:
Did you have these changes reviewed by someone?

Reply:
No, I didn't :shrug